src/makefile

```diff
@@ -9,7 +9,7 @@ endif
 
 ifeq ($(detected_OS),Darwin)        # Mac OS X
 	CXX = clang++
-	CXXFLAGS = -std=c++11 -stdlib=libc++
+	CXXFLAGS = -std=c++11 -stdlib=libc++ -g
```

Comment:
enables debug information so that gdb can have line numbers etc.
we should add production/devel/staging build options eventually and this should be excluded from prod builds blah blah blah